mempool/mempool.go

```diff
@@ -51,7 +51,7 @@ const (
 
 	// If a vote is on a block whose height is before tip minus this
 	// amount, reject it from being added to the mempool.
-	maximumVoteAgeDelta = 1440
+	MaximumVoteAgeDelta = 1440
```

Comment:
Would prefer this not be defined here now since it's misleading given it is taken via the policy.  If anything, it would be `DefaultMaximumVoteAge`, but that really should be up to the caller to set imo.

Reply:
noted, updating.